tests/test_other.py

```diff
@@ -5017,7 +5005,6 @@ def test_module_onexit(self):
   return 14;
 }
 ''')
-    try_delete('a.out.js')
```

Comment:
Do we not need to remove it anymore? Does `do_runf` remove it?

Reply:
I should have mentioned this the the description.

These delete calls date back to a past when a test case didn't always run in a clean directory.  These days they always do.